TopMovies/DTO/GraphQL/GraphQLAllCategoriesDTO.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  GraphQLAllCategoriesDTO.swift
+//  TopMovies
+//
+//  Created by Macbook Pro  on 01.04.2021.
+//
+
+import Foundation
+
+struct GraphQLAllCategoriesDTO: Decodable {
+  let data: DataAllCategoriesDTO
+  struct DataAllCategoriesDTO: Decodable {
+    let movies: MoviesAllCategoriesDTO
+    struct MoviesAllCategoriesDTO: Decodable {
+      let nowPlaying, upcoming, topRated, popular:
+        GraphQLCategoryDTO.DataCategoryDTO.MoviesCategoryDTO.EdgesCategoryDTO
+    }
+  }
+}
```

Comment:
Let's make it a bit cleaner: 
```swift
struct GraphQLAllCategoriesDTO: Decodable {
    let data: Data; struct Data: Decodable {
      let movies: Movies; struct Movies: Decodable {
        let nowPlaying: GraphQLCategoryDTO.Data.Movies.Edges
        let upcoming: GraphQLCategoryDTO.Data.Movies.Edges
        let topRated: GraphQLCategoryDTO.Data.Movies.Edges
        let popular: GraphQLCategoryDTO.Data.Movies.Edges
      }
    }
  }
```

IMHO, using `: GraphQLCategoryDTO.Data.Movies.Edges` from different DTO may end up bad if you will decide to remove some heavy field only in one query. And that may lead to some bugs, that you will receive only when you run the app.